test/parallel/test-node-output-v8-warning.mjs

```diff
@@ -0,0 +1,30 @@
+import '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+import * as snapshot from '../common/assertSnapshot.js';
+import { describe, it } from 'node:test';
+
+function replaceNodeVersion(str) {
+  return str.replaceAll(process.version, '*');
+}
+
+describe('v8 output', { concurrency: true }, () => {
+  function normalize(str) {
+    return str.replaceAll(snapshot.replaceWindowsPaths(process.cwd()), '')
+    .replaceAll(/:\d+/g, ':*')
+    .replaceAll('/', '*')
+    .replaceAll('*test*', '*')
+    .replaceAll('*fixtures*v8*', '*')
+    .replaceAll('node --', '* --');
```

Comment:
@jahjahLemonade the idea here was to make this generic because Electron (and perhaps some other embedders) run these tests with a different executable name. As this stands, it'd fail for the same reason the `*` was initially added, unintentionally defeating its purpose. I'd like to try to find a solution for some of these snapshots that's ideally a bit less brittle, given the previous python approach seemed much more flexible and easier to make more or less specific in its comparisons